hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleContextImpl.java

```diff
@@ -446,11 +445,9 @@ private void dispatchSyntheticTxn(
         final var transaction = Transaction.newBuilder()
                 .signedTransactionBytes(signedTransactionBytes)
                 .build();
-        final var transactionBytes =
-                Bytes.wrap(PbjConverter.fromPbj(transaction).toByteArray());
         childRecordBuilder
                 .transaction(transaction)
-                .transactionBytes(transactionBytes)
+                .transactionBytes(signedTransactionBytes)
```

Comment:
Is this right?

Reply:
That's what we do in `HandleWorkflow`. And I think the code there was changed to its current form after a discussion with @tinker-michaelj